common/crypto_utils_openssl/avalon_crypto_utils/crypto_utility.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2020 Intel Corporation
```

Comment:
Shall we rename common/crypto_utils_openssl to common/crypto_utils_cpp_openssl. Name should tells about it is C++ openssl dependent python model.

Reply:
May be it's good to add a ReadMe.md in both `common/crypto_utils_openssl` and  `common/crypto_utils` . Also need to reorganize openSSL and Pythoncryptodomex files/functions such that there are common files/functions and specific files/functions/ But this could be done in another PR.